docs/src/technik/adr/adr-improve-getting-started-of-services.md

```diff
@@ -0,0 +1,46 @@
+# Verbesserung der Einsteigerfreundlichkeit durch neue Default-Werte
+
+## Status
+
+<adr-status status='accepted'></adr-status>
+
+## Kontext
+
+Wenn wir OpenSource auf Github entwickeln und somit anderen Personen die Möglichkeit geben unsere Anwendung zu verwenden,
+sollten wir dafür sorgen, dass die Einstiegshürde so gering wie möglich ist.
+
+Aktuell ist nicht standardisiert welche Datenbank wann verwendet werden wird. Somit kann es sein, dass H2 zum Einsatz kommt
+oder Oracle, abhängig vom Profil, dass beim Start gesetzt wird. H2 läuft in Memory und kann somit ohne zusätzliche
+Infrastruktur verwendet werden. Oracle hingegen benötigt einen Datenbankserver.
```

Comment:
Aktuell ist nicht standardisiert welche Datenbank wann verwendet wird. Abhängig von dem beim Start verwendeten Profil kann entweder eine H2- oder Oracle-DB verwendet werden. Während H2 ohne zusätzliche Infrastruktur in Memory läuft benötigt Oracle einen Datenbankserver.

Reply:
done mit 5d866e428215b1996d2b2cadf5b43882af5f900d